Sources/SwiftProtobufCore/Message+BinaryAdditions.swift

```diff
@@ -60,6 +60,7 @@ extension Message {
   /// Returns the size in bytes required to encode the message in binary format.
   /// This is used by `serializedData()` to precalculate the size of the buffer
   /// so that encoding can proceed without bounds checks or reallocation.
+  @usableFromInline
   internal func serializedDataSize() throws -> Int {
```

Comment:
Should we consider renaming this with a shim in Compat?

Reply:
It's `internal`, so if we want to rename it I don't think the shim in Compat is necessary?